src/index.js

```diff
@@ -1,127 +1,92 @@
-; (() => {
+const MAIN_DATA = [
+    {
+        number: 1,
+        characters: ""
+    },
+    {
+        number: 2,
+        characters: "ABC"
+    },
+    {
+        number: 3,
+        characters: "DEF"
+    },
+    {
+        number: 4,
+        characters: "GHI"
+    },
+    {
+        number: 5,
+        characters: "JKL"
+    },
+    {
+        number: 6,
+        characters: "MNO"
+    },
+    {
+        number: 7,
+        characters: "PQRS"
+    },
+    {
+        number: 8,
+        characters: "TUV"
+    },
+    {
+        number: 9,
+        characters: "WXYZ"
+    },
+    {
+        number: "*",
+        characters: ""
+    },
+    {
+        number: 0,
+        characters: "+"
+    },
+    {
+        number: "#",
+        characters: ""
+    }
+]
 
-    const appElement = document.createElement("div")
-    appElement.id = "app"
-    document.body.appendChild(appElement)
+const FOOTER_DATA = [
+    {
+        name: "Favourites"
+    },
+    {
+        name: "Recents"
```

Comment:
If object is having only one field, so maybe instead of using object you could use element